src/routes/w.py

```
@@ -11,55 +11,72 @@
-        name=basename,
-        size=len(content),
-        creator=current_user.id
-    )
+    w_meta = WHTMLMetadata(name=basename, size=len(content), creator=current_user.id)
     session.add(w_meta)
-    try: session.commit()
+    try:
+        session.commit()
     except IntegrityError:
         session.rollback()
         raise HTTPException(409, detail="unique field exists")
```

Ruff (0.13.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)